coqpp/coqpp_parse.mly

```diff
@@ -465,4 +465,4 @@ doc_gram_gram_token:
 | LPAREN doc_gram_gram_tokens RPAREN { GSymbParen $2 }
 | LBRACKET doc_gram_rules RBRACKET { GSymbProd $2 }
 | STRING { GSymbString $1 }
-;
\ No newline at end of file
+;
```

Comment:
You'd have to squash this change with the earlier commit that changed this file to actually make the linter happy.

Reply:
>You'd have to squash this change with the earlier commit that changed this file to actually make the linter happy.

This is incorrect, the end of file newline lint is not per-commit.

Reply:
OK, thanks for the correction.